src/debug.c

```diff
@@ -771,7 +771,7 @@ void debugCommand(client *c) {
         addReplyStatus(c, d);
         sdsfree(d);
     } else if (!strcasecmp(c->argv[1]->ptr, "digest-value") && c->argc >= 2) {
-        /* DEBUG DIGEST-VALUE key key key ... key. */
+        /* DEBUG DIGEST-VALUE key key ... key. */
```

Comment:
This repository uses a handful of different ways to describe runs of the same token. I've picked one for consistency, but it isn't the one I would have naturally chosen, it just feels closest to the middle-of-the-road for this repository.